src/index.ts

```diff
@@ -62,6 +65,9 @@ const parseArgs = (args: string[]) =>
       case "--write":
         acc.write = true;
         break;
+        case "-r":
+        case "--rename-only-child":
+          acc.renameOnlyChild = true
```

Comment:
https://github.com/benawad/destiny/pull/89#discussion_r386085311
```suggestion
        case "-S":
        case "--avoid-single-file":
          acc.avoidSingleFile = true
```